src/pdbi.rs

```diff
@@ -161,7 +160,6 @@ pub struct StreamName<'n> {
 /// objects.
 #[derive(Debug)]
 pub struct StreamNames<'s> {
-    buf: ParseBuffer<'s>,
```

Comment:
I guess this was flagged as a dead store? Assuming the lifetime here takes care of everything, and the `ParseBuffer` is not needed to be kept alive for other kinds of reasons…

Reply:
`ParseBuffer` is a wrapper around a slice, so we don't need to keep it alive since it doesn't own data.